roles/smallfile-bench/templates/workload_job.yml.j2

```diff
@@ -40,24 +40,21 @@ spec:
           - name: uuid
             value: "{{ uuid }}"
           - name: test_user
-            value: "{{ test_user }}"
+            value: "{{ test_user | default("ripsaw") }}"
           - name: clustername
             value: "{{ clustername }}"
-{% if elasticsearch.server is defined %}
           - name: es
-            value: "{{ elasticsearch.server }}"
+            value: "{{ elasticsearch.server | default() }}"
           - name: es_port
-            value: "{{ elasticsearch.port }}"
+            value: "{{ elasticsearch.port | default() }}"
           - name: es_index
-            value: "{{ es_index }}"
-{% endif %}
+            value: "{{ elasticsearch.es_index | default("ripsaw-smallfile") }}"
           - name: redis_host
             value: "{{ bo.resources[0].status.podIP }}"
           - name: clients
             value: "{{ smallfile.clients }}"
           args:
             - python /tmp/smallfile/subscriber {{bo.resources[0].status.podIP}};
-              export TMPDIR=/tmp
```

Comment:
Are we removing the TMPDIR definition outright? In fio it was re-defined elsewhere but here it just seems to be a remove line

Reply:
@dry923 , good question.  He probably removed it because TMPDIR isn't referenced anywhere in this file, but subprocesses can reference it too.      in this line in smallfile code, TMPDIR is referenced, but if it's not defined smallfile just uses /var/tmp/, which does always exist, so not a fatal error.   And it doesn't use it for files it accesses during the test, just logging and thread-ready synch. files, so I don't think it matters.

Reply:
Exactly, that's the reason :). I checked smallfile's source and verified it. TMPDIR is back again as env var. I don't want to break any consistency of the workloads, just reorganize and clean-up.